pkg/sync/queue_manager.go

```diff
@@ -183,3 +183,25 @@ func (qm *QueueManager) RemoveRepository(repo *v1alpha1.Repository) {
 	repoKey := repoKey(repo)
 	delete(qm.queueMap, repoKey)
 }
+
+func (qm *QueueManager) QueuedPipelineRuns(repo *v1alpha1.Repository) []string {
```

Comment:
is this code only used in the test?

Reply:
right now yes
but I do think we can use in code for concurrency so will be a bit easier for debugging
and would be helpful